tests/conftest.py

```diff
@@ -54,6 +55,7 @@
 from taipy.config.common.scope import Scope
 from taipy.config.config import Config
 
+client = Client()
```

Comment:
WHy do we need that in tests, but not in the "real life"?

Reply:
@jrobinAV regarding initializing the Client() object, I have found some references that can help us determine if we need to initialize it
https://stackoverflow.com/questions/61424359/do-i-need-to-initialize-the-client-when-using-dask-locally

Reply:
yes that s sounds great!